lib/pure/os.nim

```diff
@@ -1224,12 +1232,17 @@ proc fileNewer*(a, b: string): bool {.rtl, extern: "nos$1", noNimScript.} =
 
 proc getCurrentDir*(): string {.rtl, extern: "nos$1", tags: [], noNimScript.} =
   ## Returns the `current working directory`:idx:.
+  ## This is similar to Linux, Mac, BSD, Windows PowerShell ``pwd`` terminal command.
+  ## This is the path where the Nim-compiled binary is *run*.
```

Comment:
"where the Nim-compiled binary is *run*" implies that the current working directory is always the directory where the binary is, but the cwd/pwd can be changed, so this sentence isn't correct and should be removed to avoid confusion.